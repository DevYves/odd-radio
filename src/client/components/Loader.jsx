import React, {Component} from 'react';

class Loader extends Component {
  render(){
    if (this.props.audioLoad) {
      return(
      <div>
        <i className="fa fa-spinner fa-pulse fa-2x fa-fw"></i>
      </div>
      );
    } else {
      return(
        <div></div>
      );
    }
  }
}

export default Loader;