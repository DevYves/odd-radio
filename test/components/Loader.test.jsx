import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

// React 16 Enzyme adapter
Enzyme.configure({ adapter: new Adapter() });

import Loader from '../../src/client/components/Loader';

describe('Loader', () => {

  const subject = <Loader />;

  it('matches snapshot', () => {
    expect(renderer.create(subject)).toMatchSnapshot();
  });
});