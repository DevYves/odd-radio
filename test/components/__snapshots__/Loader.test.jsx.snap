// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader matches snapshot 1`] = `
<div>
  <i
    className="fa fa-spinner fa-pulse fa-2x fa-fw"
  />
</div>
`;
